l10n/css.json

```
@@ -1618,38 +1660,44 @@
     "en-US": "table-caption elements",
     "fr": "éléments qui sont des légendes de tableaux",
     "ja": "表のキャプション要素",
-    "ru": "заголовки таблиц"
+    "ru": "заголовки таблиц",
+    "zh-CN": "表格说明文字元素"
```

https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/caption

```suggestion
    "zh-CN": "表格标题元素"
```